cmd/flags.go

```diff
@@ -126,8 +127,19 @@ func addInputFlag(cmd *cobra.Command) {
 
 // Lets this command specify an Attestation Server Address.
 func addAsAddressFlag(cmd *cobra.Command) {
-	cmd.PersistentFlags().StringVar(&asAddress, "asAddr", "https://confidentialcomputing.googleapis.com",
-		"Attestation Service address")
+	cmd.PersistentFlags().StringVar(&asAddress, "verifier-endpoint", "https://confidentialcomputing.googleapis.com",
+		"the attestation verifier endpoint used to retrieve an attestation claims token")
+}
+
+// Lets this command enable Cloud logging
+func addCloudLoggingFlag(cmd *cobra.Command) {
+	cmd.Flags().Bool("cloud-log", false, "cloud logging switch")
```

Comment:
This should be consistent with our other flags and use a global var like `cloudLog`:
cmd.Flags().Bool(&cloudLog, "cloud-log", false, "cloud logging switch")

Reply:
The function signature for cmd.Flags().Bool is func (*pflag.FlagSet).Bool(name string, value bool, usage string) *bool. It is different from cmd.PersistentFlags().StringVar(). So we could not use the pattern of  cmd.Flags().Bool(&cloudLog, "cloud-log", false, "cloud logging switch").

Reply:
Sorry I meant cmd.Flags().BoolVar. See https://github.com/google/go-tpm-tools/blob/b22bad0e47403f90d754253de3927b3edea29613/cmd/root.go#L35